### DISCUSSION TITLE
Is it possible to link more than one AAD tenant to a single outline instance?

### DISCUSSION THREAD

**Discussion by rraineygc** [outline/outline#3662] [General]
Where an organisation may have users residing across more than one AAD tenant, is it possible to allow access via the Microsoft SSO to multiple tenants?

Currently using email invite, but becomes difficult to manage

**Comment by tommoor** [comment]
Technically possible but there is no UI at this time, you'd have to write the rows into the `authentication_providers` table directly using the values in there as a guide. It's simple enough, but does need that direct db access.
